integration_test/cloudbuild-integration-test.yaml

```diff
@@ -47,13 +55,23 @@
   id: wait-for-adapter
   args:
   - ash
-  - integration_test/wait_for_port.sh
+  - integration_test/wait-for-port.sh
   - ${_ADAPTER_RUN_STEP}
   - ${_ADAPTER_PORT}
   waitFor:
-  - build-store-scu
+  - build-tools
```

Comment:
Shouldn't this wait for adapter-main instead of build-tools? (not that it matters too much)

Reply:
'waitFor:' means waiting for docker container running the step to close with success result. That's not what I need, I need adapter and storescp up and ready to execute queries.